src/python/pants/backend/docker/util_rules/docker_build_context_test.py

```diff
@@ -788,14 +788,15 @@ def test_create_docker_build_context() -> None:
             copy_source_paths=(),
             copy_build_args=DockerBuildArgs.from_strings(),
             from_image_build_args=DockerBuildArgs.from_strings(),
-            version_tags=("base latest", "stage1 1.2", "dev 2.0", "prod 2.0"),
+            # Stage without tags tests regression of #22108
```

Comment:
Is this acceptable or should regressions always be tested in a separate test?

Reply:
Fine by me!